source/common/http/http3/well_known_names.h

```diff
@@ -9,8 +9,8 @@ namespace Http {
 
 class QuicCodecNameValues {
 public:
-  const std::string Client = "client_codec";
-  const std::string Server = "server_codec";
+  const std::string Client = "quiche_client";
```

Comment:
We shouldn't let "quiche" leak into any core code. This should only be inside the extension itself, right? Can you move this into the extension directory or just remove this file entirely unless it needs to be referenced elsewhere?

Reply:
I think we don't want any core code depends on extension code. That's why I add them here. These are used to create quic codec in CodecHelper and integration test, we can't remove them.

Reply:
This is in the common directory. Just move this file into the extension directory then? There shouldn't be any reference to quiche in common code.